MH4U Dex/MonsterOverviewViewController.m

```diff
@@ -0,0 +1,43 @@
+//
+//  MonsterOverviewViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/6/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "MonsterOverviewViewController.h"
+
+#import "DamageZonesCollectionViewController.h"
+
+@interface MonsterOverviewViewController ()
+
+@property (weak, nonatomic) IBOutlet UIView *damageZonesCollectionView;
+@property (weak, nonatomic) IBOutlet UIImageView *monsterIconImage;
+@property (weak, nonatomic) IBOutlet UILabel *monsterNameLabel;
+
+@end
+
+@implementation MonsterOverviewViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.monsterNameLabel.text = self.monster;
+    NSString *imageName = [NSString stringWithFormat:@"%@-icon", [self.monster lowercaseString]];
```

Comment:
If the image name for a monster is always going to be structured this way, consider moving at least the image name, if not the whole image, into a readonly property on the model.